app/controllers/documents_controller.rb

```diff
@@ -56,7 +56,9 @@ def document_related_policies
   end
 
   def find_unpublishing
-    Unpublishing.from_slug(params[:id], document_class)
+    unpublishing = Unpublishing.from_slug(params[:id], document_class)
+
+    unpublishing unless unpublishing.try(:edition).try(:deleted?)
```

Comment:
All this `try` business is a bit annoying but there do seem to be a fair number of orphaned `unpublishing`-ses in the DB. 

```
select edition_id from unpublishings where edition_id not in (select id from editions)

97256
101747
101748
101750
101751
101752
101753
101810
113082
139367
141798
159238
160682
160811
160813
160815
166853
170555
170557
174284
174285
203445
214039
244335
244343
245034
255185
271960
271961
272183
272690
273300
276430
324740
363368
376853

```

Looks like an old issue as the last `Edition#id` is 694794 on my VM.

What do you think to clearing up the data and then we wouldn't need to try here. That would mean it would raise if `edition` was `nil` in future and we'd know there is an issue.